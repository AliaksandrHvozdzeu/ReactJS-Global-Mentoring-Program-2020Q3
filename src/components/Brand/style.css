.brand {
    height: 20%;
    display: flex;
    align-items: center;
}

.brand-title {
    color: var(--red);
    position: relative;
    font-weight: 300;
    font-size: 1.5rem;
}

.brand-title-name {
    font-weight: 800;
    text-decoration: none;
    color: var(--red);
}
