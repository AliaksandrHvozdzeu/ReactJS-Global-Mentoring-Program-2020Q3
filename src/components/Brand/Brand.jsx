import React from 'react';
import './style.css';

export default function Brand() {

  const reloadPage = () => {
    window.location.reload();
  };

  return (
    <>
      <div className="brand">
        <p className="brand-title">
          <a id="blink" className="brand-title-name" onClick={reloadPage}>netflix</a>
          roulette
        </p>
      </div>
    </>
  );

}
