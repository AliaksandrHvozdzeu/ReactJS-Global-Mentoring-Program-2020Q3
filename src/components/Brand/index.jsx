import React from 'react';
import './style.css';

const Brand = () => (
  <>
    <div className="brand">
      <p className="brand-title">
        <a className="brand-title-name" href="# ">netflix</a>
        roulette
      </p>
    </div>
  </>
)

export default Brand;
