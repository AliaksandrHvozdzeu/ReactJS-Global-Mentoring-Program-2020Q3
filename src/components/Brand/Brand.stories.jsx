import React from 'react';
import { withKnobs } from '@storybook/addon-knobs';
import Brand from './js/Brand';
import '../../App.css';

export default {
  title: 'Brand',
  decorators: [withKnobs],
};

export const Default = () => (
  <Brand />
);
