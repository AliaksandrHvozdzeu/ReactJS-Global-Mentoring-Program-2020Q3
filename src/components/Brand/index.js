import Brand from './js/Brand';

export default Brand;
