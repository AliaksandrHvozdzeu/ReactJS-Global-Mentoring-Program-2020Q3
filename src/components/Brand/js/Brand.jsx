import React from 'react';
import '../css/Brand.css';

export default function Brand() {

  const reloadPage = () => {
    window.location.reload();
  };

  return (
    <>
      <div className="brand">
        <p className="brand-title">
          <a id="blink" className="brand-title-name" href=" #" onClick={reloadPage}>netflix</a>
          roulette
        </p>
      </div>
    </>
  );

}
