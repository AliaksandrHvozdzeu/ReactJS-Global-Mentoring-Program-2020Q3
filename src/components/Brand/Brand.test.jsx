import React from 'react';
import rerender from 'react-test-renderer';
import Brand from './js/Brand';

describe('when Brand', () => {
  test('then snapshot created', () => {
    const component = rerender.create(<Brand />);
    component.root.findByType('a').props.onClick();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
