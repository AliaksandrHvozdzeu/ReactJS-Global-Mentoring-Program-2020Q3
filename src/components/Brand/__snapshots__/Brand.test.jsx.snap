// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when Brand then snapshot created 1`] = `
<div
  className="brand"
>
  <p
    className="brand-title"
  >
    <a
      className="brand-title-name"
      href=" #"
      id="blink"
      onClick={[Function]}
    >
      netflix
    </a>
    roulette
  </p>
</div>
`;
